services/pseudo_controller/src/main/java/com/futurewei/alcor/pseudo_controller/pseudo_controller.java

```diff
@@ -0,0 +1,322 @@
+/*
+This is the code for the test controller, for testing the reactions between the Network Configuration manager and
+the ACA.
+
+Params:
+1. Number of ports to generate to each aca node
+2. IP of aca_node_one
+3. IP of aca_node_two
+4. IP of the GRPC call
+5. Port of the GRPC call
+6. User name of aca_nodes
+7. Password of aca_nodes
+*/
+package com.futurewei.alcor.pseudo_controller;
+import com.futurewei.alcor.schema.*;
+import com.jcraft.jsch.Channel;
+import com.jcraft.jsch.ChannelExec;
+import com.jcraft.jsch.JSch;
+import com.jcraft.jsch.Session;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.stub.StreamObserver;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+
+public class pseudo_controller {
+
+    static String aca_node_one_ip = "ip_one";
+    static String aca_node_two_ip = "ip_two";
+    static String grpc_ip = "ip_three";
+    static int grpc_port = 123;
```

Comment:
to avoid confusions, it is better to name it as ncm_ip and ncm_port.

Reply:
fixed